globals.go

```diff
@@ -532,6 +544,18 @@ var (
 	SyncStatuses = []string{TaskSucceeded, TaskFailed}
 
 	ValidCommandTypes = []string{CommandTypeSetup, CommandTypeSystem, CommandTypeTest}
+
+	// Map from valid architectures to display names
+	ValidArchDisplayNames = map[string]string{
+		ArchWindowsAmd64: "Windows 64-bit",
```

Comment:
So apparently we build the binary on freebsd_amd64 as well, but it's not really supported elsewhere except for 1 mention in the makefile which doesn't do anything. @tychoish what do we want to do with bsd?

Reply:
If it's not displayed anywhere then it still doesn't need to be in this list

Reply:
the only engineer that used FreeBSD isn't around anymore, and there's occasionally an interest in having FreeBSD builders to test drivers things, but I can't imagine that'll get prioritized any time soon. We can remove the build entirely if you like.

Reply:
Let's change the makefile to not build on bsd then. Currently it'll end up in the list of downloads because it's a supported go architecture